draft-ietf-ice-pac.xml

```diff
@@ -191,15 +191,17 @@
       </t>
       <t>
         While the timer is running, the ICE agent MUST NOT set the state of
-        a checklist to Failed, even if the checklist has no pairs left to check.
-        As a result, the ICE agent will not remove any data streams, or set the state 
-        of the ICE session to Failed, while the timer is till running, allowing
-        received peer-reflexive candidates to be tested. 
+        a checklist to Failed, even if the checklist has no pairs left to check,
+        and MUST wait for the timer to elapse before doing so.
+        As a result, the ICE agent will not remove any data streams or set the
+        state of the ICE session to Failed until the timer elapses.
       </t>
       <t>
-        If the timer times out while the state of at least one checklist is
-        Running, the ICE agent MUST continue the ICE processing, and will eventually 
-        conclude the ICE processing following normal ICE procedures.
+        When the timer eventually elapses, the ICE agent MUST resume typical
+        ICE processing, setting any checklists containing only Failed pairs to the
+        Failed state as usual, and handling any consequences as indicated in 
+        <xref target="RFC8445" />, Section 8.1.2. Naturally, if there are no
+        such checklists, no action is necessary.
```

Comment:
In order to be aligned with 8445, instead of "Failed pairs" we should say "pairs with the state set to Failed".

Also, I suggest to say ", including setting any checklists…". Because otherwise it sounds like "typical ICE processing" and "setting the checklist states" are two different things.

Reply:
8445 uses the terminology "Failed checklist", so "Failed pairs" seems precedented; "containing only pairs in the Failed state to the Failed state" also seems clunky.

"including" sounds good to me though, will make that change.

Reply:
PTAL

Reply:
Please use consistent terminology, and talk about "timer running" instead of "timer being active".

Otherwise I am fine with the changes.

Reply:
I am fine with the text.